src/eureka/S5_lightcurve_fitting/models/AstroModel.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, model, pid=0, channel=0, eval=True):
         self.AmpCos2 = 0.
         self.AmpSin2 = 0.
         self.gamma = 0.
+        self.quasi_offset = 0.
```

Comment:
In the interest of trying to group related parameters from the same systematic model, what do you think of using the name `quasi_gamma`?